### ISSUE TITLE
Pagination in 1.0.0 fails when using permalink: false

### ISSUE THREAD

**Issue by anghelos** [11ty/eleventy#2167]
This is possibly related to #2161 and #1754 ?

**The bug**
I'm using pagination along with eleventyComputed and eleventyNavigation to create a dropdown menu with anchor links.
After updating to 1.0.0, rendering fails with below error message.
I tried updating to v1.0.1-canary.2, to check if #1956 fixed this, with no luck.

**To Reproduce**
Here's the nunjucks template:
(removing "permalink: false" fixes it, but outputs a bunch of empty pages)
```
---
pagination:
  data: dropdown
  size: 1
  alias: link
  addAllPagesToCollections: true
permalink: false
eleventyComputed:
  eleventyNavigation:
    key: "{{link.key | safe}}"
    parent: "{{link.parent | safe}}"
    url: "#{{link.key | slugify }}"
    order: "{{link.order}}"
---
```

**Expected behavior**
On Eleventy 0.12, this was rendering without a hitch.

**Error message**
```
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Writing false from ./content/dropdown_items.njk
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] > Having trouble writing template: false

`TemplateWriterWriteError` was thrown
[11ty] > The "path" argument must be of type string. Received type boolean (false)

`TypeError` was thrown:
[11ty]     TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type boolean (false)
        at new NodeError (node:internal/errors:371:5)
        at validateString (node:internal/validators:119:11)
        at Object.parse (node:path:913:5)
        at Template._write (C:\NODEJS\Eleventy\karapanou-eleventy\karapanou-eleventy\node_modules\@11ty\eleventy\src\Template.js:817:36)        
        at C:\NODEJS\Eleventy\karapanou-eleventy\karapanou-eleventy\node_modules\@11ty\eleventy\src\Template.js:923:23
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async Promise.all (index 0)
        at async Promise.all (index 56)
        at async Eleventy.executeBuild (C:\NODEJS\Eleventy\karapanou-eleventy\karapanou-eleventy\node_modules\@11ty\eleventy\src\Eleventy.js:986:13)
```

**Environment:**

- OS and Version: Windows 11
- Eleventy Version: 1.0.0

**Comment by zachleat** [comment]
I took a stab at fixing something similar, can you test with version 1.0.1-canary.2?

**Comment by anghelos** [comment]
It’s not very clear in my post, but I do mention that I tried that to no avail, unfortunately. From: Zach LeathermanSent: Thursday, January 13, 2022 7:14 PMTo: 11ty/eleventyCc: Anghelos Coulon; AuthorSubject: Re: [11ty/eleventy] Pagination in 1.0.0 fails when using permalink: false (Issue #2167) I took a stab at fixing something similar, can you test with version 1.0.1-canary.2?—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by anghelos** [comment]
As a reply to a probably deleted comment: I tried updating to v1.0.0-canary.2, with no change.

**Comment by zachleat** [comment]
Reproduced this one 🏆

**Comment by zachleat** [comment]
This fix is a bit riskier, so it won’t be included with 1.0.1 but will ship with 2.0.0-canary.5
